pkg/server/apiHandlerv2.go

```diff
@@ -672,3 +677,308 @@ func (h *apiHandlerv2) uuid(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	w.Write(cloudEnvelope)
 }
+
+func extractQuoteAttestTemplate(quote *attest.ZAttestQuote) *common.PCRTemplate {
+	template := common.PCRTemplate{
+		GPSInfo: quote.GpsInfo,
+	}
+	for _, el := range quote.GetPcrValues() {
```

Comment:
Do you get these in numeric order? Hard to tell whether there is a golang map in there resulting in random order.

Reply:
Looks like it is slice, not map https://github.com/lf-edge/eve/blob/ec150ab5359f8af63f651024d18d22a0580df055/api/go/attest/attest.pb.go#L978.

Reply:
It might be worth adding that here in a comment, so that the next person to look at this does not get confused.